ui/pages/institutional/interactive-replacement-token-page/index.scss

```diff
@@ -0,0 +1,23 @@
+.interactive-replacement-token-page {
+  opacity: 0.8;
+
+  &__item {
+    border-bottom: 1px solid #d2d8dd;
```

Comment:
Can you use the color from `src/css/design-tokens.css`?
maybe `--color-border-default` this one